rastervision_pytorch_learner/rastervision/pytorch_learner/learner.py

```diff
@@ -715,7 +715,8 @@ def numpy_predict(self, x: np.ndarray, raw_out: bool = False) -> Any:
         x = torch.tensor(x)
         x = self.to_batch(x)
         x = x.permute((0, 3, 1, 2))
-        out = self.predict(x, normalize=True, raw_out=raw_out)
+        out = self.predict(
```

Comment:
It might make sense to cast `x` to type float (float32) in `predict` when normalization is not done because it seems that `x` must be the same type as the weights.

Reply:
Sounds good